x-pack/test/functional/apps/ml/data_visualizer/index_data_visualizer.ts

```diff
@@ -144,7 +144,7 @@ export default function ({ getService }: FtrProviderContext) {
     nonMetricFieldsFilter: 'airline',
     nonMetricFieldsTypeFilter: 'keyword',
     expected: {
-      totalDocCountFormatted: '34,415',
+      totalDocCountFormatted: '86,274',
```

Comment:
The fact that the tests are passing now with the changed numbers indicates that saved searches are not respected correctly when loading the data (the full data set has 86274 docs, but the saved search that is loaded here, should only show 34415 of them). If saved searches are still supported with the new index data visualizer, I'd suggest to revert this test change and fix the data loading on the app side.

Reply:
My tests confirm what @pheyos is seeing here, and the saved search doesn't seem to be used in the new page. In the old page, you can see the search in the search bar on opening:

![image](https://user-images.githubusercontent.com/7405507/102091823-3b00a500-3e17-11eb-87e3-ae5c69d49fb5.png)

We should still support loading from a saved search.

Reply:
This saved search issue should now be fixed, I have also reverted the changes here https://github.com/elastic/kibana/pull/85726/commits/958b660e3b151b77001a6dc7bb460b5ae278a84f and here https://github.com/elastic/kibana/pull/85726/commits/c33a88f087611e282de6d01b35432bf5d5a00204